src/statespace/SE2.cpp

```diff
@@ -152,7 +152,6 @@ StateSpace::State* SE2::create_state() {
 }
 
 double SE2::normalize_angle_rad(const double& theta_rad) const {
-    assert(m_bins % 2 == 0);
```

Comment:
Should be a separate PR; but regardless this function should be using the function from utils